docs/static/includes/sample.telem.tf

```
@@ -1,5 +1,60 @@
 locals {
-  module_version = "v1.0.0"
+  # This is the unique id for the module that is supplied by the AVM team.
+  # TODO: change this to the PUID for the module. See https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-sfr3---category-telemetry---deploymentusage-telemetry
+  telem_puid = "00000000"
+
+  # TODO: change this to the name of the module. See https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-sfr3---category-telemetry---deploymentusage-telemetry
+  module_name = "CHANGEME"
+
+  # Should be either `res` or `ptn`
+  module_type = "res"
+
+  # This is an empty ARM deployment template.
+  telem_arm_template_content = <<TEMPLATE
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {},
+  "variables": {},
+  "resources": [],
+  "outputs": {
+    "telemetry": {
+      "type": "String",
+      "value": "For more information, see https://aka.ms/avm/telemetry"
```

```suggestion
      "value": "For more information, see https://azure.github.io/Azure-Verified-Modules/help-support/telemetry/"
```

The other shortlink goes elsewhere today